tests/test_transformers.py

```diff
@@ -44,6 +46,7 @@ def test_transformations_new_size(transformer, extra_size):
         Noise(0.1),
         Tsne(2, n_iter=250),
         OpenTsne(2, n_iter=100),
+        Ivis(2, k=10, batch_size=10),
```

Comment:
If you're listing hyperparams here that you consider important, it might be valid to upgrade them from `**kwargs` to normal keyword arguments.